silver/tests/spec/test_invoice.py

```diff
@@ -61,8 +61,8 @@ def test_post_invoice_without_invoice_entries(self):
             "number": None,
             "provider": "http://testserver/providers/1/",
             "customer": "http://testserver/customers/1/",
-            "archived_provider": {},
-            "archived_customer": {},
+            "archived_provider": None,
```

Comment:
Because the tests not passed. The returned values were with 'None' and not with '{}' see with drone exec.